src/pages/videoListing/VideoListing.jsx

```diff
@@ -16,13 +33,11 @@ export function VideoListing() {
 
         <div className="video-grid-wrapper">
           <div className="video-grid">
-            <VideoCard />
-            <VideoCard />
-            <VideoCard />
-            <VideoCard />
-            <VideoCard />
-            <VideoCard />
-            <VideoCard />
+            {data.map((video) => (
```

Comment:
Just a suggestion: I think u can put one check condition here also to avoid errors if data is empty.